source/options.tsx

```diff
@@ -57,6 +20,13 @@ function moveDisabledFeaturesToTop(): void {
 }
 
 function buildFeatureCheckbox({id, description, screenshot}: FeatureMeta): HTMLElement {
+	const descriptionElement = domify.one(description) as HTMLParagraphElement;
```

Comment:
It isn't needed, but a non-null assertion operator is needed:

```suggestion
	const descriptionElement = domify.one(description)!;
```

I made it explicit though, to highlight that the parsed description is wrapped in a `<p>`.

Reply:
Other than in this PR, it doesn’t seem particularly useful to know it’s a `p` rather than a generic element. It’s actual type is determined by the parser anyway, so this type could be wrong here, which is why I try to avoid assertions.

Reply:
Okay, I dropped the type assertion.